lua/obsidian/commands/follow_link.lua

```diff
@@ -45,7 +45,7 @@ return function(client, data)
         if confirmation == "y" or confirmation == "yes" then
           -- Create a new note.
           local aliases = name == location and {} or { name }
-          note = client:new_note(location, nil, nil, aliases)
+          note = client:new_note(location, nil, vim.fn.expand "%:p:h", aliases)
```

Comment:
I recently added this field on the client. The result is the same but I think this is a little cleaner:

```suggestion
          note = client:new_note(location, nil, client.buf_dir, aliases)
```

Could you also replace the `vim.fn.expand` call in `link_new.lua` so we're consistent?

Reply:
Excellent, idea. It is done!